index.js

```diff
@@ -36,7 +36,9 @@ class WebpackReactComponentNamePlugin {
       factory.hooks.parser.for('javascript/auto').tap('WebpackReactComponentNamePlugin', parser => {
         parser.hooks.program.tap("WebpackReactComponentNamePlugin", ast => {
           // Ignore dependency files
-          if (parser.state.current.resource == null || parser.state.current.resource.indexOf("node_modules") !== -1 || !VALID_FILE_SUFFIXES_REGEX.test(parser.state.current.resource.toLowerCase())) {
+          if (parser.state.current.resource == null 
+            || (!this.options.parseDependencies && parser.state.current.resource.indexOf("node_modules") !== -1) 
```

Comment:
This is causing the unit tests to fail because `this.options` can be null / undefined. I'll push up an update that ensures that `this.options` is never null / undefined.